nextcloud_update.sh

```diff
@@ -309,7 +309,16 @@ fi
 # Update all Nextcloud apps
 if [ "${CURRENTVERSION%%.*}" -ge "15" ]
 then
-    occ_command app:update --all
+    UPDATED_APPS=$(occ_command app:update | sed '1~2d' | awk '{print $1}' | tr '\n' " ")
+    UPDATED_APPS=${UPDATED_APPS// /", "}
```

Comment:
What this does is, it deletes every second line from the output, prints the first element every line and substitues the linebreaks through spaces. Afterwards all spaces get substituted by ", " which should result in the following formatting:
app1, app2, app3

Reply:
This unfortunately doesn't work yet. We probably have to append the output of occ_command to a file. Do you have an idea how to get all of this working for a file?
afterwards the result has to get put into a variable so that we can check whether we can send the notification

Reply:
Basically what I wanted to do was getting the app names from the output and posting them to the admins.
But if that is too complicated maybe it is enough to post to the admins, that app updates got installed. What do you think?

Reply:
Please forget what I have said. It is working